t/proj/settings.py

```diff
@@ -77,12 +82,28 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'postgres',
+        'USER': 'postgres',
+        'HOST': 'localhost',
+        'PORT': '5432',
+        'OPTIONS': {
+            'connect_timeout': 1000,
+        }
+    },
+    'secondary': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'postgres',
```

Comment:
Since your testing settings have two connections but to the same database, how do your tests verify that it wrote to the correct one?  Shouldn't you specify two different table names?

Reply:
We verify that we wrote the database here: https://github.com/celery/django-celery-results/pull/63/files#diff-5083bf592c9b0d766e45e8c37e83db8eR98

The goal of this change is to allow updating the same database within a transaction.

Reply:
Ah, I see now.  Neat.